### DISCUSSION TITLE
Suggestion: Have `meow-reverse` run `C-x C-x` instead of `exchange-point-and-mark`?

### DISCUSSION THREAD

**Discussion by okamsn** [meow-edit/meow#393] [Ideas]
That way, it would also work with `rectangle-mark-mode`, which binds `rectangle-exchange-point-and-mark` to `C-x C-x`, matching `exchange-point-and-mark`.

This would match the pattern of having `meow-save` running `M-w` and `meow-kill` running `C-w`, which already work with `rectangle-mark-mode`.

**Comment by eshrh** [comment]
Seems good to me. Would you like to open a PR or should I implement it?

**Reply by okamsn** [reply]
I have created the PR in #394.
